Survey/Sources/Presentation/Modules/Home/Views/SurveyItemView.swift

```diff
@@ -14,6 +14,7 @@ struct SurveyItemView: View {
     @State private var fadeInOut = false
 
     let survey: Survey
+    let didNext: () -> Void
```

Comment:
I think we should name this `willGoToDetail` or presentSurveyDetail or something else. What  do you think?